src/main/java/com/zerobase/fintech/auth/security/AuthenticationFilter.java

```diff
@@ -0,0 +1,54 @@
+package com.zerobase.fintech.auth.security;
+
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.ObjectUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class AuthenticationFilter extends OncePerRequestFilter {
+
+    private final TokenProvider tokenProvider;
+
+    @Value("${spring.jwt.prefix}")
+    private String tokenPrefix;
+
+    @Value("${spring.jwt.header}")
+    private String tokenHeader;
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
+        String token = resolveTokenFromRequest(request);
+
+
+        // 토큰 유효성 검사
+        if(StringUtils.hasText(token) && this.tokenProvider.validateToken(token)){
```

Comment:
```suggestion
        if (StringUtils.hasText(token) && this.tokenProvider.validateToken(token)) {
```
coding convnetion 에 어긋나는 if 문에 대한 indent 이네요. 현업에서는 coding convention 에 대해 굉장히 중요하게 생각하니, 이번 프로젝트를 통해 익혀두시구 습관화해보시는 것도 좋을 것 같아요.
https://naver.github.io/hackday-conventions-java/#space-between-keyword-parentheses